### PR TITLE
Added command reference for GetLit CLI

### PR DESCRIPTION
**Pull request by furkanakal**
# Description

Even though command table was provided in the CLI docs, getting started intro was lacking. I modified it so that it has step-by-step moving forward guide now.

NOTE: It is not complete yet, PR opened on purpose of providing feedback regarding the writing style, format, and etc.

### CODE REVIEW

File: docs/tools/getlit-cli.md [hunk 1]
```diff
@@ -33,4 +35,105 @@ To use the GetLit CLI, navigate to a directory or your existing project, and the
...
+ */
+ 
+// This will exceed the default file size limit
+// import * as LitJsSdk from "@lit-protocol/lit-node-client-nodejs";
+ 
+type SignData = number[];
+ 
+const helloWorld: SignData = [
+  72, 101, 108, 108, 111, 32, 87, 111, 114, 108, 100,
+];
```
Comment (joshLong145) [inline docs/tools/getlit-cli.md:95]: we should change this to use a `Text Encoder` with `crypto.subtle` as we know sign hashes not raw messages.

File: docs/tools/getlit-cli.md [hunk 2]
```diff
@@ -33,4 +35,105 @@ To use the GetLit CLI, navigate to a directory or your existing project, and the
...
+ 
+(async () => {
+  // this requests a signature share from the Lit Node
+  // the signature share will be automatically returned in the HTTP response from the node
+  const sigShare = await LitActions.signEcdsa({
+    toSign: new Uint8Array(helloWorld),
```
Comment (joshLong145) [inline docs/tools/getlit-cli.md:101]: see above

File: docs/tools/getlit-cli.md [hunk 3]
```diff
@@ -33,4 +35,105 @@ To use the GetLit CLI, navigate to a directory or your existing project, and the
...
+    publicKey, // <-- You should pass this in jsParam
```
Comment (joshLong145) [inline docs/tools/getlit-cli.md:102]: perhaps make this some constant like `<your pkp publickey>` so users know they have to fill it in.

**Comment by furkanakal** [comment]
Thanks @joshLong145 for this valuable feedback. Which parts exactly do you suggest me to replace with some links to the existing guides you mentioned above?

**Comment by joshLong145** [comment]
> Thanks @joshLong145 for this valuable feedback. Which parts exactly do you suggest me to replace with some links to the existing guides you mentioned above?

We already have breakdowns of how to perform various operations in lit actions you can reference the lit action section of our docs to get an understanding of what is already covered. 
https://developer.litprotocol.com/v3/sdk/serverless-signing/quick-start
So we should keep this section just to what this tool is doing, not what PKPs / Lit Actions are in general.

**Comment by joshLong145** [comment]
Is this completed? while this is a helpful explanation it lacks much of the cli functionality.

**Comment by furkanakal** [comment]
When we talked last time, I thought it was okay to you. I haven't written something about other commands as either they have similar things with the rest of the doc, or they are super clear by their name. Let me write something for each command concisely.

I'll remove package-lock.json.

**Review by joshLong145 — APPROVED** [review]

**Comment by glitch003** [comment]
Thanks, merging!

**Comment by glitch003** [comment]
or wait, not merging until i check with you @furkanakal

**Comment by glitch003** [comment]
or wait, not merging until i check with you @furkanakal

**Review by glitch003 — APPROVED** [review]

**Pull request merged**